components/SelectInventoriedResource.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import {gql, useQuery} from "@apollo/client";
+import {useAuth} from "../lib/auth";
+
+type Resource = { id: string, name: string, onhandQuantity: any }
```

Comment:
is onhandQuantity comes from graphql? cause onHandQuantity should be more suitable as a name!